environs/imagedownloads/simplstreams.go

```diff
@@ -0,0 +1,214 @@
+// Package imagedownloads implements image-downloads metadata from
+// simplestreams.
+// Copyright 2016 Canonical Ltd.
+// Licensed under the AGPLv3, see LICENCE file for details.
+package imagedownloads
```

Comment:
Line break :)

Also i think we want:
// copyright statement
// line break
// package description
// line break
//package declaration

I have also seen package, e.g. ~/state/lease where package description was in a separate doc.go file \o/ I thought it was neat :) and clear where to find it when needed...